src/bluesky/protocols.py

```diff
@@ -464,12 +464,16 @@ def stop(self, success=True) -> SyncOrAsync[None]:
 @runtime_checkable
 class Subscribable(HasName, Protocol):
     @abstractmethod
-    def subscribe(self, function: Callback) -> None:
+    def subscribe(self, function: Callback, run: bool = True) -> None:
```

Comment:
```suggestion
    def subscribe(self, function: Callback, *, run: bool = True) -> None:
```

Do we want to make this keyword-only?